include/exadg/time_integration/time_step_calculation.h

```diff
@@ -253,6 +253,8 @@ calculate_time_step_cfl_local(
   CFLConditionType const                                         cfl_condition_type,
   MPI_Comm const &                                               mpi_comm)
 {
+  // Need to update ghost values in the case of continuity constraints.
+  velocity.update_ghost_values();
```

Comment:
This is currently done always (L2 and HDIV). But we could also call this function in 

`IncNS::SpatialOperatorBase<dim, Number>::calculate_time_step_cfl()`

where we could easily distinguish between L2 and HDIV, because the spatial operator knows these parameters. Would such a design be clearer in terms of single responsibility?